app/Community/Controllers/UserCommentController.php

```diff
@@ -4,29 +4,49 @@
 
 namespace App\Community\Controllers;
 
-use App\Community\Actions\AddCommentAction;
 use App\Community\Actions\GetUrlToCommentDestinationAction;
+use App\Community\Concerns\IndexesComments;
+use App\Community\Data\CommentData;
+use App\Community\Data\UserCommentsPagePropsData;
 use App\Community\Enums\ArticleType;
 use App\Community\Requests\StoreCommentRequest;
+use App\Data\PaginatedData;
+use App\Data\UserData;
 use App\Models\Comment;
 use App\Models\User;
 use App\Models\UserComment;
+use App\Policies\UserCommentPolicy;
 use Illuminate\Contracts\View\View;
 use Illuminate\Http\JsonResponse;
 use Illuminate\Http\RedirectResponse;
 use Illuminate\Http\Request;
+use Inertia\Response as InertiaResponse;
 
 class UserCommentController extends CommentController
 {
-    public function index(User $user): View
-    {
-        $this->authorize('viewAny', [UserComment::class, $user]);
-
-        $comments = UserComment::paginate();
+    use IndexesComments;
 
-        return view('user.comment.index')
-            ->with('user', $user)
-            ->with('grid', $comments);
+    public function index(User $user): InertiaResponse|RedirectResponse
+    {
+        return $this->handleCommentIndex(
+            commentable: $user,
+            policy: UserComment::class,
+            routeName: 'user.comment.index',
+            routeParam: 'user',
+            view: 'user/[user]/comments',
+            createPropsData: function ($user, $paginatedComments, $isSubscribed, $me) {
+                return new UserCommentsPagePropsData(
+                    targetUser: UserData::fromUser($user)->include('id'),
+                    paginatedComments: PaginatedData::fromLengthAwarePaginator(
+                        $paginatedComments,
+                        total: $paginatedComments->total(),
+                        items: CommentData::fromCollection($paginatedComments->getCollection())
```

Comment:
I get the following error accessing any user's comments list.
```
Attempted to lazy load [commentable] on model [App\Models\UserComment] but lazy loading is disabled.
```
It seems to be coming from this line.

Reply:
Looks like it was an issue in a policy. Fixed in latest.